docs/test.md

```diff
@@ -0,0 +1,34 @@
+# Test
+
+Most classes have unit tests (classname.spec.ts), though we also want to test the whole code in an "End-To-End" way.
+These tests are written in a separate test-folder, which can be found on the same level as src.
+
+To run tests, use the following command:
+``` bash
+ # to test all .spec files
+ npm run test
+ 
+ # to test all .spec files from a certain app / lib
+ npx nx test <app|lib>.
+```
+
+## How do we write tests?
```

Comment:
Die Info, dass wir mit Jest testen wäre noch gut, dann musst du auch nicht im Detail erklären, das wir describe und it verwenden. Das ergibt sich dann aus dem Kontext

Reply:
wollte des nicht zweimal erwähnen, is ja im normalen dev.md schon enthalten, aber gut das nehm ich das wieder rein